mmv1/products/memorystore/Instance.yaml

```diff
@@ -0,0 +1,309 @@
+---
+!ruby/object:Api::Resource
+base_url: "projects/{{project}}/locations/{{location}}/instances"
+create_url: "projects/{{project}}/locations/{{location}}/instances?instanceId={{instance_id}}"
+self_link: "projects/{{project}}/locations/{{location}}/instances/{{instance_id}}"
+id_format: "projects/{{project}}/locations/{{location}}/instances/{{instance_id}}"
+import_format:
+  - "projects/{{project}}/locations/{{location}}/instances/{{instance_id}}"
+name: Instance
+description: A Google Cloud Memorystore instance.
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+min_version: beta
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  encoder: templates/terraform/encoders/memorystore_instance.go.erb
```

Comment:
It looks like you may not need these (unless this part is still under development)

Reply:
added